src/main/java/org/dominokit/domino/ui/forms/SuggestBox.java

```diff
@@ -107,6 +117,10 @@ public String getValue() {
         return getInputElement().asElement().value;
     }
 
+    public void setSuggestBoxStore(SuggestBoxStore store) {
```

Comment:
can we also have a builder style method for this, lets keep the set and make a new method that calls the setter, something like `public SuggestBox suggestStore(SuggetStore)`

Reply:
There are already two setters (`setLoaderEffect()` and `setType()`) with return type `SuggestBox`. To be consequent I would prefer to fit the setter `setSuggestBoxStore` also with return type `SuggestBox`. WDYT?

Reply:
go ahead. i agree